src/components/ImportExport.js

```diff
@@ -145,7 +146,12 @@ class ImportExport extends React.Component {
             }
           } else {
             const hydrant = processHydrant(feature, index);
-            newHydrants[hydrant.get('id')] = hydrant;
+
+            //Only Allow Hydrants With ids
+            if(hydrant.get('id')){
+              newHydrants[hydrant.get('id')] = hydrant;
+            }
```

Comment:
This will make it so hydrants without a description field that it can parse an ID from aren't added to the store, right? 

It's good because it means things won't break if they try to load random kml's without that field, but it also means ALL external formats won't work, since none of them would have a description field with that format. 

Probably the best call for now, just noting it (and making sure I'm not missing anything).